tests/functional/adapter/simple_seed/test_simple_seed.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from dbt.tests.adapter.seed.test_seed import (
+    BasicSeedTests,
+    SeedConfigFullRefreshOff,
```

Comment:
Snowflake does not support the truncation of downstream tables on a full refresh. Hence, I've omitted the SeedConfigFullrefreshOn test